index.html

```diff
@@ -195,35 +254,42 @@ <h2>CONTACT US</h2>
                 <br />
                 <input type="email" id="email" name="email" placeholder="jeremy@gmail.com" required /><br />
                 <br />
```

Comment:
- [ ] As the previous reviewer mentioned, you need to fix this design flow, kindly make sure the name and email inputs are aligned properly so your form is designed properly 

![Screenshot_20220708_042900](https://user-images.githubusercontent.com/25227022/177912346-3ccf99f6-3031-4536-947e-7d0d31aa9670.png)

Reply:
Hey @Ismailco 
 
Thanks for your review.

My question is that I have fixed this error and it works perfectly in my advice, would you mind checking once again and tell me which screen size works perfectly in all screen sizes on my side thanks.